src/components/GridFeatured/GridFeatured.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="grid-featured">
+        <block-featured
+            v-for="(block, i) in blocks"
+            :key="i"
+            :to="block.uri"
+            :is-news="block.isNews"
+            :has-hover="block.hasHover"
```

Comment:
This will actually be a method, based on the `postType` from the API I think, or maybe the GQL `__typename`, because sometimes they link to a page or a post. So can you have this call a method `hasHover(block)` and that just returns `return block.hasHover`for now and I'll update once the API is built.